website/app/controllers/show.js

```diff
@@ -9,12 +9,32 @@ import { inject as service } from '@ember/service';
 import { showdownConfig } from '../shared/showdown-config';
 
 export default class ShowController extends Controller {
+  queryParams = [
+    {
+      selectedTab: 'tab',
+    },
+    'searchQuery',
+    'selectedIconSize',
+  ];
+
   @service fastboot;
 
   @tracked sections = A([]);
   @tracked tabs = A([]);
   @tracked tocs = A([]);
 
+  get selectedTabIndex() {
+    if (!this.selectedTab) {
+      return 0;
+    }
+
+    let tab = this.tabs.find((el) => {
+      // trim trailing slashes from query param
+      return el.label.toLowerCase() === this.selectedTab.replace(/\/$/, '');
```

Comment:
this is so it's `tab=library` vs `tab=Library` (because that's how i had all the tests written). happy to change this and update the tests if keeping things lowercase in the URL seems of low value

Reply:
the button inside the tab has this HTML in the DOM:
```
<button id="tab-guidelines" class="doc-text-navigation" role="tab" aria-controls="guidelines" aria-selected="" type="button">Guidelines</button>
```

maybe we can use the `id` of the tab, so we can use it then to identify the element more easily? (I know there would be a duplication, `tab=tab-guidelines` but maybe is not that of a problem?)

Reply:
that's what I initially used but having `tab-` as part of the query param felt off to me. in practice behind the scenes it's not any easier or harder, so sticking with the label seems slightly cleaner to me.